PROJECT_CHARTER.md

```diff
@@ -0,0 +1,81 @@
+# Fastify Charter
+
+The Fastify project aims to build a fast and low overhead web framework for Node.js.
+
+
+## Section 0: Guiding Principles
+
+The Fastify project is part of the [OpenJS Foundation][openjs foundation], which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+
+## Section 1: Scope
+
+Fastify is a web framework highly focused on providing the best developer experience with the least overhead and a plugin architecture.
+
+### 1.1: In-scope
+
++ Develop a web framework for Node.js with a focus on developer experience, performance and extensibility.
++ Plugin Architecture
++ Support web protocols
++ Official plugins for common user requirements
++ Documentation:
+  + Project (policies, processes, and releases)
+  + Guides and Tutorials
+  + Framework API
+  + Website
++ Write easier APIs for developers
++ Tools:
+  + CI services
+  + Bots to improve overall efficiency
+  + Releases
++ Support:
+  + Community
+  + Users's issues and questions
++ Contributors's pull request review
+
+### 1.2: Out-of-Scope
+
++ Support versions of Node.js at EOL (end of life) stage
++ Support serverless architecture
++ Contributions that violates the [Code of Conduct](CODE_OF_CONDUCT.md)
+
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+Technical leadership for the projects within the [OpenJS Foundation][openjs foundation] is delegated to the projects through their project charters by the [OpenJS Foundation Cross-Project Council](https://openjsf.org/about/governance/) (CPC). In the case of the Fastify project, it is delegated to the [Fastify's Team](README.md#team) (FTM). The OpenJS Foundation's business leadership is the Board of Directors (the "Board").
+
+This Fastify Charter reflects a carefully constructed balanced role for the FTM and the CPC in the governance of the OpenJS Foundation. The charter amendment process is for the FTM to propose changes using simple majority of the full FTM, the proposed changes being subject to review and approval by the CPC. The CPC may additionally make amendments to the FTM charter at any time, though the CPC will not interfere with day-to-day discussions, votes or meetings of the FTM.
+
+ex. [Node.js TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-2-evolution-of-openjs-foundation-governance)
```

Comment:
I think this line's been left here accidentally from the charter template?

Reply:
Good spot, we'll fix!